RadixWallet/Features/HomeFeature/Coordinator/Home+View.swift

```diff
@@ -208,15 +207,6 @@ private extension View {
 			content: { NPSSurvey.View(store: $0) }
 		)
 	}
-
-	private func relinkConnector(with destinationStore: PresentationStoreOf<Home.Destination>) -> some View {
-		sheet(
```

Comment:
This used to be a sheet, and now is a full screen (which is more invasive in my opinion). Maybe we can have `SheetOverlayCoordinator` as well?

Reply:
Maybe better to have a new `OverlayReducer.Destination.State` say `sheet` and maybe rename the `FullScreenOverlayCoordinator`, but yes it’s a nice improvement. I’ll investigate what it implies.